### ISSUE TITLE
zero losses during training

### ISSUE THREAD

**Issue by qilbster** [zzzxxxttt/pytorch_simple_cornernet#7]
Hi,

I've been trying to run the model on a different dataset, also with labelled bounding boxes. Im currently at the model training stage, where I can see the losses at each step. So far I'm still within the first epoch, but the losses are for the most part zero. They sometimes register a non-zero value but often go fluctuate back to zeros. Here's an example.
![Screenshot from 2021-03-16 13-13-19](https://user-images.githubusercontent.com/78620619/111314672-87978e80-8659-11eb-936b-52db088125f4.png)

I was just wondering if this behaviour is normal. Are there any potential issues that could lead to this behaviour? The dataset I'm working with tends to have multiple bounding boxes per image. The background class is dominant but there's not an extreme imbalance. 

If you have any ideas why this could be happening it would be much appreciated. Also, thanks for the code! Its really useful for our project :)

**Comment by zzzxxxttt** [comment]
I think this behavier is not normal, zero loss means no sufficient gradient to train the network. One possible reason is that the random crop always extracts image patches with no object, so maybe disable random crop will help.

**Comment by qilbster** [comment]
thanks for the reply. I was worried about that too. will try that
